### PR TITLE
ConsoleInteraction: Catch ValueError

### PR DESCRIPTION
**Pull request by andreidinu**
### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [ ] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

Fixes https://github.com/coala/coala/issues/2997

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
hey, please run coala over your code, thanks :)

**Comment by aptrishu** [comment]
Hey @andreidinu ,
your Circle CI tests are failing, because you have not added tests for the code you have added in ConsoleInteraction.py.
Also go to [this link ](https://coala.io/commit) to learn `how to write good commits`. :+1:

**Comment by aptrishu** [comment]
Also mark the checkboxes in the checklist given above they show the reviewers how ready your PR is.

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -311,6 +311,12 @@ def print_results_formatted(log_printer,
                 'Unable to print the result with the given format string.',
                 exception)
 
+        except ValueError as exception:
```
Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: This need to be tested in `ConsoleInteractionTest`

**Comment by Nosferatul** [comment]
closing due to high inactivity

**Pull request closed without merging**